src/js/experimental/media-playback-rate-listbox.js

```diff
@@ -0,0 +1,77 @@
+import MediaChromeListbox from './media-chrome-listbox.js';
+import { window, document } from '../utils/server-safe-globals.js';
+import { MediaUIAttributes, MediaUIEvents } from '../constants.js';
+
+const slotTemplate = document.createElement('template');
+slotTemplate.innerHTML = `
+  <style>
+    media-chrome-listitem {
+      white-space: var(--media-captions-listbox-white-space, nowrap);
+    }
+
+  </style>
+`;
+
+const compareTracks = (a, b) => {
+  return a.label === b.label && a.language === b.language;
+}
+
+class MediaPlaybackrateListbox extends MediaChromeListbox {
+  static get observedAttributes() {
+    return [
+      ...super.observedAttributes,
+      'aria-multiselectable',
+      MediaUIAttributes.MEDIA_PLAYBACK_RATE
+    ];
+  }
+
+  constructor() {
+    super({slotTemplate});
+  }
+
+  attributeChangedCallback(attrName, oldValue, newValue) {
+    if (attrName === MediaUIAttributes.MEDIA_PLAYBACK_RATE && oldValue !== newValue) {
+      this.value = newValue;
```

Comment:
this is the magic that updates the selected item when the playback rate changes.